setup.py

```diff
@@ -42,112 +37,26 @@ def get_sphinx_command():
-                + ", ".join(e.name for e in self.extensions)
-            )
-
-        cmake_version = LooseVersion(
-            re.search(r"version\s*([\d.]+)", out.decode()).group(1)
-        )
-        if cmake_version < "3.3.0":
-            raise RuntimeError("CMake >= 3.3.0 is required")
-
-        for ext in self.extensions:
-            self.build_extension(ext)
-
-    def build_extension(self, ext):
-        extdir = osp.abspath(osp.dirname(self.get_ext_fullpath(ext.name)))
-        extdir = osp.join(extdir, ext.name)
-        cmake_args = [
-            "-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=" + extdir,
-            "-DPYTHON_EXECUTABLE=" + sys.executable,
-        ]
-
-        cfg = "Debug" if self.debug else "Release"
-        build_args = ["--config", cfg]
-
-        if platform.system() == "Windows":
-            cmake_args += [
-                "-DCMAKE_LIBRARY_OUTPUT_DIRECTORY_{}={}".format(cfg.upper(), extdir)
-            ]
-            if sys.maxsize > 2 ** 32:
-                cmake_args += ["-A", "x64"]
-            build_args += ["--", "/m"]
-        else:
-            cmake_args += ["-DCMAKE_BUILD_TYPE=" + cfg]
-            build_args += ["--", "-j{}".format(max(1, os.cpu_count() - 3))]
-
-        env = os.environ.copy()
-        env["CXXFLAGS"] = '{} -DVERSION_INFO=\\"{}\\"'.format(
-            env.get("CXXFLAGS", ""), self.distribution.get_version()
-        )
-        if not osp.exists(self.build_temp):
-            os.makedirs(self.build_temp)
-        subprocess.check_call(
-            ["cmake", ext.sourcedir] + cmake_args, cwd=self.build_temp, env=env
-        )
-        subprocess.check_call(
-            ["cmake", "--build", "."] + build_args, cwd=self.build_temp
-        )
-
-        # copy nmodl module with shared library to extension directory
-        copy_tree(os.path.join(self.build_temp, 'nmodl'), extdir)
-
-class NMODLTest(test):
-    """Custom disutils command that acts like as a replacement
-    for the "test" command.
-
-    It first executes the standard "test" command, then runs the
-    C++ tests and finally runs the "doctest" to also validate
-    code snippets in the sphinx documentation.
-    """
-
-    def distutils_dir_name(self, dname):
-        """Returns the name of a distutils build directory"""
-        dir_name = "{dirname}.{platform}-{version[0]}.{version[1]}"
-        return dir_name.format(
-            dirname=dname, platform=sysconfig.get_platform(), version=sys.version_info
-        )
+        self.run_command("thisCommandShouldNotExist")
 
-    def run(self):
-        super().run()
-        subprocess.check_call(
-            [
-                "cmake",
-                "--build",
-                os.path.join("build", self.distutils_dir_name("temp")),
-                "--target",
-                "test",
-            ]
-        )
 
 
-install_requirements = ["jinja2>=2.9.3", "PyYAML>=3.13", "sympy>=1.3"]
+install_requirements = [
+    "PyYAML>=3.13",
+    "sympy>=1.3",
```

Comment:
```suggestion
    "sympy>=1.3,,1.6",
```

Reply:
you mean:

`"sympy>=1.3,<1.6"`

right?